core/src/main/java/io/grpc/internal/CensusTracingModule.java

```diff
@@ -382,7 +382,8 @@ public ServerStreamTracer newServerStreamTracer(String fullMethodName, Metadata
       // Safe usage of the unsafe trace API because CONTEXT_SPAN_KEY.get() returns the same value
       // as Tracer.getCurrentSpan() except when no value available when the return value is null
       // for the direct access and BlankSpan when Tracer API is used.
-      final ClientCallTracer tracerFactory = newClientCallTracer(CONTEXT_SPAN_KEY.get(), method);
+      final ClientCallTracer tracerFactory =
+          newClientCallTracer(ContextUtils.getValue(Context.current()), method);
```

Comment:
Note `ContextUtils.getValue()` will always return non-null, while previously `CONTEXT_SPAN_KEY.get()` may return null.

Reply:
The returned `Span` eventually goes to `spanBuilderWithExplicitParent()`. Would it behavior the same with `null` vs the non-null value returned by `ContextUtils.getValue()`?

Reply:
Yes the behavior will be the same. Instead of returning `null` previously, `ContextUtils.getValue()` now returns a blank (invalid) span. Both `null` and invalid span are handled in the same way in `spanBuilderWithExplicitParent()`: https://github.com/census-instrumentation/opencensus-java/blob/7b9ea2251c63852bf5e0633d402600c90822de04/impl_core/src/main/java/io/opencensus/implcore/trace/SpanBuilderImpl.java#L76.